src/postgres-util/src/schemas.rs

```diff
@@ -39,6 +40,13 @@ pub async fn publication_info(
     client: &Client,
     publication: &str,
 ) -> Result<Vec<PostgresTableDesc>, PostgresError> {
+    let server_version_num = client
+        .query_one("SHOW server_version_num", &[])
```

Comment:
Naively wondered if this is guaranteed to return (and return a parseable string) across different PostgreSQL managed services — looking at Neon and Supabase, it seems like it is!

Reply:
Yep, `server_version_num` is the machine parsable one, and unless someone does something really weird when compiling Postgres will be an integer representation of the Postgres version! `server_version` is the one that's human readable and thus has a fair bit of variance across providers.